examples/with-apollo/yarn.lock

```diff
@@ -0,0 +1,3163 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@arunoda should they be removed from all examples? Saw one in the `with-mobx` example too.